mathicsscript/settings.m

```diff
@@ -0,0 +1,14 @@
+
+
+Settings`$ShowFullForm::usage = "If this variable is set to True, it makes that mathicsscript shows the FullForm of the input before evaluate"
+
+Settings`$PygmentsStyle::usage = "When ShowFullForm is True, and PygmentsStyle is set, makes that mathicsscript shows the FullForm using highlight with pygments"
```

Comment:
This is comment and default value are not quite right. PygmentsStyle colors all output, not just the FullForm display when that is set. 

Try changing this from say "colorful" to "inkpot" and you will see a change in display. 

And that leads to the second thing that is weird is setting this to False, boolean since this is normally a String.

Reply:
Thinking about what to do for Settings`$PygmentsStyle more, maybe this should be commented out so that it can be set by the user, but not set by default here.

In the code we detect if we have a dark background or not, and set the style according to that if none was otherwise set.

If it is to be set in a settings file like this then probably adding a setting for COLORFGBG would be helpful too, it is set by [term-background](https://pypi.org/project/term-background/) and the main function that is used is called `is_dark_background()` which might also be exposed inside Mathics.

Reply:
> Thanks for doing. The description and default setting for Settings`PygmentsStyle are a little weird though. See comment below.

Yes, I just write something to state how to proceed. We can improve that part once we decide what should do each variable.

Reply:
What I didn't realize is that we also need to change the settings of the terminal object. It seems that to do this, we need to create a new object. Is there a more efficient way to do this? Or is something that can be done on each call to print?

Reply:
Not a more efficient way that I can think of easily. And also it is not a big deal. People don't generally change pygments styles settings, and creating a new one more like a dozen milliseconds when human interaction time is on the order of half of a second.